rkt/pods.go

```diff
@@ -914,8 +915,8 @@ func (p *pod) getContainerPID1() (pid int, err error) {
 			return -1, err
 		}
 
-		if !os.IsNotExist(err) || !p.isRunning() {
-			return -1, err
+		if !p.isRunning() {
+			return -1, fmt.Errorf("pod %v is not running after refreshState()", p.uuid)
```

Comment:
"after refreshState()"  is an implementation detail. Can you remove it from the message?

> pod %v is not running anymore

If it is called from rkt/enter.go:runEnter, it would be:

> unable to determine the pid for pod %q: pod %v is not running anymore